caos.ansible_roles/roles/cleanup/tasks/package-Windows.yaml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Get infra agent productID
+  ansible.windows.win_powershell:
+    script: (Get-WmiObject win32_product | where{$_.name -eq "{{ item }}"}).IdentifyingNumber
+  register: packages_info
+  loop: "{{ packages_to_cleanup }}"
+
+- name: Uninstall infra agent
```

Comment:
```suggestion
- name: Uninstall package
```

?

Reply:
Nice 👁️ Done in d832c855867ff293db534e296aed06c9009ef371